src/test/java/com/quadfour/QuadFour/TaskServiceTest.java

```diff
@@ -95,24 +86,9 @@ private void thenTaskIsSaved() {
             fail();
         }
     }
+    private void thenQuadFourReturnsZeroResults() {
+		assertEquals(0, tasks.size());
+	}
 
-    //	private void whenTheUserSearchesForCercis() throws Exception {
```

Comment:
Good idea.  Remove commented out code entirely.  Instead of commenting out code, my suggestion is:
1) Use feature branches in Git to keep experimental work separate from master.
2) If you're removing something that used to be there, just remove it entirely.  You can always go back in history to see what used to be there.